lib/ohmqtt/ohmqtt.h

```diff
@@ -26,9 +23,9 @@ class ohmqtt {
   ESP8266WiFiClass* _WiFi;
   MQTT_CALLBACK_SIGNATURE;
   MQTT_RESUBSCRIBE_SIGNATURE;
-  const char* client_id;
```

Comment:
I removed const as this means it can only be set once, which were set to be null